relayer/src/connection.rs

```diff
@@ -1062,9 +1074,11 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Connection<ChainA, ChainB> {
     pub fn build_conn_confirm_and_send(&self) -> Result<IbcEvent, ConnectionError> {
         let dst_msgs = self.build_conn_confirm()?;
 
+        let tm = TrackedMsgs::new(dst_msgs, "create connection");
```

Comment:
Also lots of places using "create connection". Should they all be refactored to a helper method that would do it in one place?

Reply:
I think it may actually be wrong to call this "create connection", and instead we should be more accurate, and also provide the concrete name of the handshake step, in this case it would be: "create connection confirm". Agree with the refactoring ideea.